wordpress-mobile/WordPress-Android
# Create ViewModel in themes module and link bottom sheet

2/3 Part of #15014

## Fixes
This PR adds a ViewModel in the themes module (where there wasn't any previously). And links the theme activation action sheet with it. 

## To test:

1. Open the app
2. Scroll down and select Themes under the Look and Feel
3. Scroll to your desired theme
4. Press the three dots(…) button
5. Press Activate
6. Verify that the new dialog appears with the correct theme name in the title, in information text below the title, and in the option "Use {theme name}'s homepage layout"

**NOTE:** activate and preview buttons on the action sheet do not do anything yet.

## Regression Notes

**Potential unintended areas of impact**
This changes very little in the existing codebase, which is opening the bottom sheet from `ThemeBrowserActivity` and even there this change is isolated.

**What I did to test those areas of impact (or what existing automated tests I relied on)**
N/A

**What automated tests I added (or what prevented me from doing so)**
I added `ThemesViewModelTest` testing the viewModel I added.

**PR submission checklist:**


**Depends on:** https://github.com/wordpress-mobile/WordPress-Android/pull/15029
